src/main.lua

```diff
@@ -971,6 +971,7 @@ addEventingHandler(
 		local fundFrom = msg.Tags["Fund-From"]
 		local checkAssertions = function()
 			assert(type(msg.Tags.Name) == "string", "Invalid name")
+			assert(msg.Tags.Quantity, "Must provide a quantity")
```

Comment:
Nit: This is sort of already considered on the next line. Consider simply adding `(msg.Tags.Quantity and ...` to the line below if nil-ness is the problem.

Reply:
Without this line we throw an aos type error below

Reply:
you can modify to account for both

```suggestion
			assert( msg.Tags.Quantity and 
				tonumber(msg.Tags.Quantity) > 0
					and tonumber(msg.Tags.Quantity) < 9990
					and utils.isInteger(msg.Tags.Quantity),
				"Invalid quantity. Must be an integer value greater than 0 and less than 9990"
			)```

Reply:
8c31c61